zerver/lib/bugdown/api_code_examples.py

```diff
@@ -115,7 +206,7 @@ def run(self, lines: List[str]) -> List[str]:
                         else:
                             text = self.render_fixture(function)
                     elif key == 'example':
-                        if argument == 'admin_config=True':
+                        if argument == 'admin_config=True' and language == "python":
```

Comment:
I'm not sure this change is required?

Reply:
It's required because `render_curl_example` doesn't take `admin_config` as an argument.

Reply:
Why is `admin_config` set as an argument in that case, then?

Reply:
Oh, right. Sorry about that. I'm making the change :+1:.